### PR TITLE
OAK-10304: log registration of invalid namespace names

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java [hunk 1]
```diff
@@ -90,4 +86,37 @@ protected void refresh() {
...
+            List<String> myLogs = customLogs.getLogs();
+            assertEquals(1, myLogs.size());
+            assertTrue(myLogs.get(0).contains("Registering invalid namespace name 'example.com' for prefix 'foo', please see"));
+        }
+        finally {
+            customLogs.finished();
+        }
+    }
+
+    private NamespaceRegistry getNamespaceRegistry(ContentSession session, Root root) {
```
Comment (anchela) [inline oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java]: that could be static, no?

File: oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java [hunk 2]
```diff
@@ -29,10 +31,13 @@
 import org.apache.jackrabbit.oak.OakBaseTest;
 import org.apache.jackrabbit.oak.api.ContentSession;
 import org.apache.jackrabbit.oak.api.Root;
+import org.apache.jackrabbit.oak.commons.junit.LogCustomizer;
 import org.apache.jackrabbit.oak.fixture.NodeStoreFixture;
 import org.apache.jackrabbit.oak.spi.security.OpenSecurityProvider;
 import org.junit.Test;
 
+import ch.qos.logback.classic.Level;
```
Comment (anchela) [inline oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java]: is that import covered by a test-dependency in the pom?

**Review by anchela — APPROVED** [review]
hi @reschke , looks good... i just have 2 minor comments but didn't verify it in an IDE just wondering from looking at the diff.

File: oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java (hunk 1 above)
Comment (reschke) [reply oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java]: yes.

File: oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java (hunk 2 above)
Comment (reschke) [reply oak-it/src/test/java/org/apache/jackrabbit/oak/plugins/name/ReadWriteNamespaceRegistryTest.java]: No, and it should be from sl4j. Thanks for spotting.

**Pull request merged**